### PR TITLE
fix: typo in docs

### PR DESCRIPTION
**Pull request by MyaLongmire**
### Required for all PRs:

- [ ] Updated associated README.md.
- [ ] Wrote appropriate unit tests.
- [ ] Pull request title or commits are in [conventional commit format](https://www.conventionalcommits.org/en/v1.0.0/#summary)

resolves #

Removes a link that leads to a 404 (I went through the file history and it never linked to anything that exsisted) and removes the section of the table as that was the only entry

### CODE REVIEW

**Review by powersj — APPROVED** [review]

File: plugins/parsers/xpath/README.md [hunk 1]
```diff
@@ -8,12 +8,12 @@ For supported XPath functions check [the underlying XPath library][xpath lib].
 
 ## Supported data formats
 
-| name                                    | `data_format` setting | comment |
-| --------------------------------------- | --------------------- | ------- |
-| [Extensible Markup Language (XML)][xml] | `"xml"`               |         |
-| [JSON][json]                            | `"xpath_json"`        |         |
-| [MessagePack][msgpack]                  | `"xpath_msgpack"`     |         |
-| [Protocol buffers][protobuf]            | `"xpath_protobuf"`    | [see additional parameters](protocol-buffers-additiona-settings)|
```
Comment (reimda) [inline plugins/parsers/xpath/README.md:16]: @srebhan did you mean to include another doc for protobuf back when you added support?

Comment (srebhan) [reply plugins/parsers/xpath/README.md:16]: To be honest this should refer to the subsection below (line 18) but it seems I do have a typo there (`protocol-buffers-additiona-settings` vs the correct `protocol-buffers-additional-settings`)... :-(

**Comment by srebhan** [comment]
@MyaLongmire it's a typo and therefore dead (see my comment to @reimda's comment). Could you maybe just fix the link instead of removing it?

File: plugins/parsers/xpath/README.md [hunk 2]
```diff
@@ -13,7 +13,7 @@ For supported XPath functions check [the underlying XPath library][xpath lib].
 | [Extensible Markup Language (XML)][xml] | `"xml"`               |         |
 | [JSON][json]                            | `"xpath_json"`        |         |
 | [MessagePack][msgpack]                  | `"xpath_msgpack"`     |         |
-| [Protocol buffers][protobuf]            | `"xpath_protobuf"`    | [see additional parameters](protocol-buffers-additiona-settings)|
+| [Protocol buffers][protobuf]            | `"xpath_protobuf"`    | [see additional parameters](protocol-buffers-additional-settings)|
```
Comment (sspaink) [inline plugins/parsers/xpath/README.md:16]: I think this requires a `#` at the beginning (https://stackoverflow.com/questions/6695439/how-to-link-to-a-named-anchor-in-multimarkdown)

```suggestion
| [Protocol buffers][protobuf]            | `"xpath_protobuf"`    | [see additional parameters](#protocol-buffers-additional-settings)|
```

Comment (MyaLongmire) [reply plugins/parsers/xpath/README.md]: That explains why when i was trying `protocol-buffers-additional-settings` I was still getting a 404 :)

**Review by sspaink — APPROVED** [review]

**Pull request merged**